### PR TITLE
release notes

### PR DESCRIPTION
**Pull request by VladimirWork**
Signed-off-by: Vladimir Shishkin <vladimir.shishkin@dsr-corporation.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -48,6 +50,66 @@
...
+| indy-plenum | 1.10.0 |
+| indy-node | 1.10.0 |
+| sovrin | 1.1.58 |
+
+### Additional Information:
+**There are possible OOM issues during 3+ hours of target load or large catch-ups at 8 GB RAM nodes pool so 32 GB is recommended.**
+**PBFT View Change was implemented but not enabled so old View Change is active now.**
+
+### Major Changes
+- PBFT View Change implementation
```
Comment (ashcherbakov) [inline CHANGELOG.md]:
```suggestion
- PBFT View Change implementation (not enabled yet) and corresponding code improvements 
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -48,6 +50,66 @@
...
+- BLS fixes and improvements
```
Comment (ashcherbakov) [inline CHANGELOG.md]:
```suggestion
- BLS multi-signature fixes and improvements
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -48,6 +50,66 @@
...
+- Stability fixes
+
```
Comment (ashcherbakov) [inline CHANGELOG.md]: Please mention ZMQ bump as well in Major Changes

File: CHANGELOG.md (hunk 1 above)
Comment (VladimirWork) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (VladimirWork) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 3 above)
Comment (VladimirWork) [reply CHANGELOG.md]: Done.

**Comment by VladimirWork** [comment]
test this please

**Pull request merged**